render_profiles.py

```diff
@@ -0,0 +1,125 @@
+#!/bin/env python3
+
```

Comment:
A bigger block comment at the top would help to better explain the purpose of this script